xarray/tests/test_dataset.py

```diff
@@ -3900,6 +3900,16 @@ def test_shift(self):
         with raises_regex(ValueError, 'dimensions'):
             ds.shift(foo=123)
 
+    def test_shift_multidim(self):
```

Comment:
`test_roll_multidim` ?  Just for ease of finding it later.

Reply:
Thanks. It's my mistake. Corrected.